dnplab/dnpIO/topspin.py

```diff
@@ -386,18 +381,17 @@ def topspin_jcamp_dx(path):
     return attrs
 
 
-def topspin_vdlist(path, expNum):
+def topspin_vdlist(path):
     """
     Return topspin vdlist
 
     Args:
         Path (str): Directory of data
-        expNum (int): Experiment number
 
     Returns:
         numpy.ndarray: vdlist as numpy array
     """
-    fullPath = path + str(expNum) + "/vdlist"
+    fullPath = path + "vdlist"
```

Comment:
do you need `os.path.join` ?

Reply:
I fixed this in many more places as well